patches/minecraft/net/minecraft/enchantment/EnchantmentHelper.java.patch

```diff
@@ -1,6 +1,21 @@
 --- ../src-base/minecraft/net/minecraft/enchantment/EnchantmentHelper.java
 +++ ../src-work/minecraft/net/minecraft/enchantment/EnchantmentHelper.java
-@@ -302,7 +302,7 @@
+@@ -143,11 +143,14 @@
+         return field_77520_b.field_77497_a;
+     }
+ 
++
```

Comment:
This newline should not be added

Reply:
Sorry, will remove that
My git diff highlighting is always confused by all the +- in the patch files :angry: